frameworks/digital-outcomes-and-specialists-5/questions/services/accessibleApplicationsOutcomes.yml

```diff
@@ -1,15 +1,17 @@
 name: Designing and building accessible applications
-question: Will you design and build accessible applications that meet the 2018 accessibility regulations?
+question: >
+  Will you design and build accessible applications that meet the 2018 accessibility regulations?
 hint:
-  Read guidance about how to meet the regulations and <a target="_blank" rel="noopener noreferrer" href="https://www.gov.uk/guidance/accessibility-requirements-for-public-sector-websites-and-apps">design and build accessible services</a>.
+  Read guidance about how to meet the regulations and <a target="_blank" rel="noopener noreferrer" href="https://www.gov.uk/guidance/accessibility-requirements-for-public-sector-websites-and-apps">design and build accessible services (opens new tab)</a>.
 type: boolean
 required_value: true
 
 depends:
   - "on": "lot"
     being:
       - digital-outcomes
+      - digital-specialists
```

Comment:
We were asked to make this change: https://docs.google.com/spreadsheets/d/1FNGwe56myW2ddO6XQUtex209TJQ2Ub99PTzcWxpejsQ/edit#gid=866429329&range=D55

Reply:
Can we remove the duplicate question for each specialists to avoid asking the same question twice?

```
$ find questions/services -name '*AccessibleApplications.yml'
questions/services/contentDesignerAccessibleApplications.yml
questions/services/webOperationsAccessibleApplications.yml
questions/services/technicalArchitectAccessibleApplications.yml
questions/services/designerAccessibleApplications.yml
questions/services/qualityAssuranceAccessibleApplications.yml
questions/services/developerAccessibleApplications.yml
questions/services/serviceManagerAccessibleApplications.yml
```

Reply:
Ah - good catch. I hadn't noticed that some of the specialists had their own questions. Removed.